### ISSUE TITLE
Handle A1/REF/ALT from plink2 glm

### ISSUE THREAD

**Issue by ofrei** [precimed/python_convert#44]
To handle plink2 glm results we may need something like this:
```
df['A2'] = [(ref if (alt==a1) else alt) for ref, alt, a1 in zip(df['REF'].values, df['ALT'].values, df['A1'].values)]
```
Sample from ``plink2 glm`` output:
```
REF ALT	A1
A	G	G	
C	T	C	
T	G	G	
C	A	C	
C	T	T	
T	C	T	
G	A	G	
G	A	A	
G	A	G	
```

**Comment by denvdm** [comment]
Yeah, to pile on to this, would be nice to have sumstats.py recognize plink2 standard column names. Currently, besides the REF and ALT allele columns, #CHROM, and ID are not interpreted/deleted. Of course I know how to manually override this, but given plink2 is now used more and more, good to make this automatic?
